<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="legendCheck">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Legend</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="Size">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>64</width>
     <height>33</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QgsFieldExpressionWidget" name="expField1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QgsColorButtonV2" name="colorButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QgsDataDefinedButton" name="dataDefined">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>30</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="LayerCombo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>64</width>
     <height>33</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSlider" name="alpha">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QgsFieldExpressionWidget" name="expField2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QgsFieldExpressionWidget" name="expField3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButtonV2</class>
   <extends>QToolButton</extends>
   <header>qgscolorbuttonv2.h</header>
  </customwidget>
  <customwidget>
   <class>QgsDataDefinedButton</class>
   <extends>QToolButton</extends>
   <header>qgsdatadefinedbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldExpressionWidget</class>
   <extends>QWidget</extends>
   <header>qgsfieldexpressionwidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LayerCombo</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>expField1</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LayerCombo</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>expField2</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>alpha</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LayerCombo</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>expField3</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
